tests/test_indexer.py

```diff
@@ -28,7 +28,8 @@
 import dss
 from dss import Config, DeploymentStage
 from dss.config import IndexSuffix
-from dss.events.handlers.index import process_new_s3_indexable_object, process_new_gs_indexable_object, notify
+from dss.events.handlers.index import process_new_s3_indexable_object, process_s3_removed_object
```

Comment:
These can be imported in a single statement without exceeding the line length limit, as follows:
```
from dss.events.handlers.index import (process_new_s3_indexable_object, process_s3_removed_object,
                                       process_new_gs_indexable_object, notify)
```